content/documentation/github.md

```diff
@@ -1,101 +1,143 @@
+- The summary should be a phrase that completes the sentence “This commit will __”.
+- Use present/imperative tense, such as “Add contributor names and bios;” “Fix typos in section pages;” or “Change header color”.
+
+When naming commits, think of them as purposefully connected bundles that you could undo at a later date without effecting other areas of your project. Rather than committing all the random changes you made to a single file or everything you did before lunch that day, commit changes that *make sense together*.
 
-- In order to run and view the Quire preview of a project cloned from GitHub, you’ll need to first run `quire install` in your project directory.
+[[Include more tips / examples of good commits]]
 
+{{< q-class "box tip" >}}
+- Changes are organized by the file in which they were made. Use the checkboxes at the lefthand of your window to select which changes will or won’t be included in a given commit. Break your work into individual commits or commits with multiple changes.
 {{< /q-class >}}
 
-## Displaying Your Project Preview on GitHub
+### Push Changes
+
+Even after saving changes in your text editor and committing them in GitHub Desktop, GitHub still doesn’t know about them. For that, you need to [push](hhttps://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/pushing-changes-to-github#pushing-changes-to-github) your commits to GitHub. Use the "Push origin" link in the upper-right corner. Pushing can include one or many commits, and you can push as often as you’d like, but it’s typically good to do it at least a couple of times a day when actively working on a project.
+
+Now the changes to your project have been successfully saved in GitHub!
+
+## Collaborate with Others using GitHub
+
+One of the main advantages of using GitHub is that it's an excellent tool for managing projects what have multiple people working on it. Once the project is on GitHub and either forked or cloned by you and your colleagues, you can all make changes and commits to the same project.
+
+When setting-up a new repository you have [two options](https://docs.github.com/en/github/getting-started-with-github/types-of-github-accounts): you can you can invite users to become collaborators to your personal repository or groups of people can collaborate across many projects at the same time in organization accounts. Each has different options for [managing settings and permissions](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-user-account-settings).
+
+Clear communication is vital when collaborating with GitHub. If multiple people are working on a project and all making changes to the same files, there’s a good chance you will run into conflicts. While git is specifically designed to deal with these sorts of issues, it's important to avoid working on the same sections at the same time.
+
+### Create a Branch
```

Comment:
You're probably already planning on it, but these two sections should be expanded on. Maybe there's text in the doc you're working on for Editorial that could be used here?